subprojects/packagefiles/win-iconv/meson.options

```diff
@@ -0,0 +1,13 @@
+option(
+    'tests',
+    type: 'boolean',
+    value: true,
+    description: 'build tests',
+)
+
+option(
+    'libiconv_dll',
+    type: 'string',
+    value: '',
+    description: 'comma separated list (e.g. "iconv.dll,libiconv.dll"), this is used to get win32 dlls at runtime',
```

Comment:
This doesn't seem very clear.  Maybe something like `comma-separated list of iconv DLLs to try loading at runtime, e.g. "iconv.dll,libiconv.dll"`?

Reply:
I just copied the upstream description and added a clarification, but it is still not very clear, so I'll change it 👍🏼